aqua/DesktopManager/Portfile

```diff
@@ -24,6 +24,13 @@ worksrcdir		${name}
 
 universal_variant no
 
+pre-fetch {
+    if {${os.major} != 8} {
+        ui_error "${name} ${version} requires Mac OS X 10.4."
+        return -code error "incompatible Mac OS X version"
```

Comment:
Q: should this instead say `return -code error "incompatible macOS version"`? Should all instances of `"incompatible Mac OS X version"` be replaced with `"incompatible macOS version"` for all portfiles?

Reply:
> Q: should this instead say `return -code error "incompatible macOS version"`? Should all instances of `"incompatible Mac OS X version"` be replaced with `"incompatible macOS version"` for all portfiles?

We prefer to refer to the Mac operating system by its correct historical name where possible ("Mac OS X" or "OS X" or "macOS"), but when the message is shown to a range of OS versions, I tend to use the name of the newest affected version. So in this case for DesktopManager, I would use "incompatible macOS version". Other ports would need to be evaluated on a case-by-case basis.

Reply:
Thanks for the explanation. I have pushed a revised commit 070c991133 to use "incompatible macOS version" as you suggest.